autoscheduler/scheduler/utils.py

```diff
@@ -55,6 +56,14 @@ def __eq__(self, other):
         return (self.start_time == other.start_time and self.end_time == other.end_time
                 and self.meeting_days == other.meeting_days)
 
+class BasicOptions(enum.Enum):
```

Comment:
I don't think I like the `BasicOptions` name since I think it's too like broad / not specific enough. Maybe something like `BasicPreferences`/`FilterPreferences`/`FilterOptions`?

Reply:
maybe `BasicFilter`? I didn't want to use `FilterOptions` because you could also filter by section

Reply:
I went with BasicFilter. Lmk if you don't like it/think of something better